clients/drcachesim/common/trace_entry.h

```diff
@@ -369,6 +369,12 @@ typedef enum {
      */
     TRACE_MARKER_TYPE_PAGE_SIZE,
 
+    /**
+     * The marker value contains the ID of recorded syscalls.
```

Comment:
The plural "syscalls" makes this confusing: multiple are in a single marker?  Reword to "This marker is emitted prior to each system call when -enable_kernel_tracing is specified.  The marker value contains the system call identifier."

(Note that we may want to emit this when not doing kernel tracing too but we'll tweak the wording if we implement that.)

Reply:
Done.